docs/scim/v1/scimv1.md

```diff
@@ -0,0 +1,361 @@
+In short, SCIM makes user data more secure and simplifies the user experience by automating the user identity lifecycle management process.  With SCIM, user identities can be created in external systems, such as OKTA or Active directory.  Since SCIM is a standard, directory data (users and groups they belong to), can be stored consistently and communicated via SCIM to different applications.  This allows companies to automate the provisioning process while also maintaining a single “source of truth”.
+
+SCIM is a REST and JSON-based protocol that defines a client and server role. A client is usually an identity provider (IDP), like Okta, that contains a robust directory of user identities. A service provider (SP) like Beyond Identity, that needs a subset of information from those identities. When changes to identities are made in the IdP, including create, update, and delete, they are automatically synced to the SP according to the SCIM protocol. The IdP can also read identities from the SP to add to its directory and to detect incorrect values in the SP that could create security vulnerabilities. For end users, this means that they have seamless access to applications for which they’re assigned, with up-to-date profiles and permissions.
+
+Beyond Identity’s (BI) SCIM server implementation follows the [tenant/realm](https://developer.beyondidentity.com/docs/v1/platform-overview/architecture) paradigm.  That is, a separate SCIM server is available for each realm.
+
+The SCIM user resource is equivalent to BI identity object.  When the SCIM client creates a user, our SCIM service will create an identity.  The SCIM group resource maps to BI’s group.  The BI SCIM configuration can be inspected via standard SCIM discovery endpoints.  
+
+##Setup
+The SCIM server is available by default whenever the customer creates a new realm.  However, in order to use any SCIM functionality an access token with `scim:all` scope is required.  For more information on setting up tenants, realms, applications and obtaining access tokens with specified scope, please see [Getting Started](https://developer.beyondidentity.com/docs/v1/getting-started) and [Using Beyond Identity for Authentication](https://developer.beyondidentity.com/docs/v1/using-bi-for-auth).
+We recommend using a dedicated application for SCIM operations.  This way, the customer can differentiate between API and SCIM operations when going through the logs.
+
+##Discovery
+You can use the following endpoints to discover the implementation details of BI's SCIM server.
+
+| Endpoint                                | SCIM spec
+| :---:                                   | :---:         
+| {{baseUrl}}/scim/ServiceProviderConfig  | https://tools.ietf.org/html/rfc7643#section-5
```

Comment:
I **think** these partial URLs might look better in backticks. The ones below as well.

Reply:
They are actually not partial, since {{baseurl}} has https in it.

Reply:
It would still look better if you used back ticks I think. Want to try it out and see what it looks like?